packages/react-components/theme-designer/src/components/ColorTokens/ColorTokensList.tsx

```diff
@@ -69,14 +70,14 @@ const useStyles = makeStyles({
 });
 
 const ColorTokenRow: React.FunctionComponent<ColorTokenRowProps> = props => {
-  const { brand, brandValue, brandValueString } = props;
+  const { brand, brandValue, brandValueString, selected } = props;
   return (
     <MenuItemRadio
       icon={<CircleFilled primaryFill={brand[brandValue]} />}
       name={brandValueString}
       value={brandValueString}
     >
-      Untitled {brandValueString}
+      {selected ? <b>Untitled {brandValueString}</b> : <>Untitled {brandValueString}</>}
```

Comment:
make this a className and associated style so that its easier to change in the future

<span className={selected ? 'classes.selected' : '' }>